service/worker/migration/force_replication_workflow.go

```diff
@@ -374,41 +374,49 @@ func enqueueReplicationTasks(ctx workflow.Context, workflowExecutionsCh workflow
 	var lastActivityErr error
 
 	for workflowExecutionsCh.Receive(ctx, &workflowExecutions) {
```

Comment:
do you want to control the activity concurrency?

if so, here is a example: https://github.com/uber/cadence/blob/master/canary/concurrentExec.go

Reply:
explained offline.